chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1077,22 +1076,30 @@ async def get_unspent_lineage_info_for_puzzle_hash(_: bytes32) -> Optional[Unspe
     # 2. After skipping MAX_SKIPPED_ITEMS, we stop processing further items.
 
     async def make_and_send_big_cost_sb(coin: Coin) -> None:
+        """
+        Creates a spend bundle with a big enough cost that gets it close to the
+        maximum block clvm cost limit. The generated spend bundle has
+        `target_cost` and the expected maximum block clvm cost is
+        `max_block_clvm_cost` minus block overhead.
+        """
+        target_cost = 548_868_044
```

Comment:
as a reader of this test, it's virtually impossible to know whether this number is correct

Reply:
I added more context.